### ISSUE TITLE
Wait?

### ISSUE THREAD

**Issue by MarkMoretto** [MicrosoftDocs/VBA-Docs#504]
Why have two of basically the same examples, but none of the three examples utilizes the optional Wait argument?  I understand that it's a simple Boolean, but this is pretty much par-for-the-course with Microsoft examples.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4b69b359-8a02-a209-50bd-3cd165ca94b2
* Version Independent ID: 16a202b1-95af-680e-3f97-fccc5b4d6e30
* Content: [AppActivate statement](https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/appactivate-statement)
* Content Source: [Language/Reference/User-Interface-Help/appactivate-statement.md](https://github.com/MicrosoftDocs/VBA-Docs/blob/master/Language/Reference/User-Interface-Help/appactivate-statement.md)
* Product: **office**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by ashleedawg** [comment]
Agreed with @MarkMoretto. The two examples will confuse a newer user into thinking there is a difference between `AppID` and `ReturnValue`.  Also, the variables are only half-declared as variants, but should be `Double` (based on the result of `TypeName`).
